SoftFox/SoftFox/SoftFox.h

```diff
@@ -1,25 +1,41 @@
 #pragma once
 
 #include "resource.h"
-/*
+#include "Texture.h"
+# include "Level.h"
+
 class SoftFox
 {
 public:
-	SoftGame();
+	SoftFox();
 	~SoftFox();
 
+	void run();
+
 	static const int WINDOW_HEIGHT = 600;
 	static const int WINDOW_WIDTH = 800;
 	static const int SPRITE_SIZE = 64;
 	static const int PLAYER_MOVEMENT_SPEED = 4;
 
+	const char* level_name = "..\\Level\\Level.txt";
+
 	int getPlayerX() { return playerX; }
 	int getPlayerY() { return playerY; }
 
 private:
 	SDL_Window* window;
 	SDL_Renderer* renderer;
 
+	SDL_Texture* playerSprite;
+	SDL_Texture* platformSprite;
+
+	Level* level;
```

Comment:
There's an issue with circular includes here (SoftFox.h includes Level.h and Level.h includes SoftFox.h). This can be fixed either by putting a forward declaration to `Level` in this file, or by removing the (unused, as far as I can tell) `#include "SoftFox.h"` from Level.h.